CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to neutralinojs-cli
+First off, kudos for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to neutralinojs-cli, which is hosted as a part of [Neutralinojs](https://github.com/neutralinojs) on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## Table Of Contents
+
+- [Code of Conduct](#code-of-conduct)
+- [Making a PR](#making-a-pr)
+
+## Code of Conduct
+
+This project encourages each and everyone to contribute to it. By participating, you are expected to uphold the spirits of open source culture.
+
+## Making a PR
+
+1. Fork the latest state of the repo using git.
+2. Clone the forked repo and pull it to have latest code locally. Also, add a Git remote for the original repository.
+3. Create a feature branch in which you place your changes. Make sure all your code changes are made in your respective branch.
+4. Add proper commit messages to help others understand what you are trying to implement via your branch. Refer [this](https://www.conventionalcommits.org/en/v1.0.0/) for proper commit rules.
+5. Push the branch to GitHub. Open a pull request from the new branch to the original repo. Clean up after your pull request is merged.
+6. Once everything is reviewed, your PR will be merged. Congratulations :tada::tada: The Neutralinojs team thanks you :sparkles: Hope to send you more such :sparkles:
+
+- Commit messages must follow the following format: `[<type1, type2, ...>]: <description>`
+
+#### Type
+
+Must be one of the following:
+
+- **Cleanup**: Changes that do not affect the meaning of the code (white-space, formatting, missing
+  semi-colons, dead code removal etc.)
+- **Documentation**: Documentation only changes
+- **Feature**: A new feature
+- **Fix**: A bug fix
+- **Performance**: A code change that improves performance
+- **Refactor**: A code change that neither fixes a bug or adds a feature
+- **Tests**: Adding missing tests or fixing them
+
+
+#### Description
+
+- A short description of the changes you made in the commit.
+
+#### Example
+
+- `[Feature]: Implement Login Flow`
+- `[Documentation]: Add Code of Conduct for neutralinojs-cli`
+- `[Tests]: Write Integration tets for submodules of neutralinojs`
```

Comment:
Hey.. Thanks for sending this great documentation. I have a small request for this section. Please note that we don't follow (or ask contributors to) follow a strict commit message format. You can read the reason behind it from : https://codezri.org/docs/releasezri/intro
 So, could you please submit the markdown file without this section.

Thanks :)